scripts/docs_version_pragma_check.sh

```diff
@@ -32,13 +32,114 @@ SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-build}
 source "${REPO_ROOT}/scripts/common.sh"
 source "${REPO_ROOT}/scripts/common_cmdline.sh"
 
+function getAllAvailableVersions()
+{
+    allVersions=()
+    local prevVer
+    local currVer
+    while IFS= read -r line
+    do
+        if (echo "$line" | grep -Po '(?<=soljson-v)\d+.\d+.\d+(?=\+commit)' >/dev/null)
+        then
+            currVer=$(echo "$line" | grep -Po '(?<=soljson-v)\d+.\d+.\d+(?=\+commit)')
+            if [ "$currVer" != "$prevVer" ]
+            then
+                allVersions=( "$currVer" "${allVersions[@]}" )
+                prevVer="$currVer"
+            fi
+            # first available release on github
+            if [ $currVer == "0.4.11" ]
+            then
+                break
+            fi
+        fi
+    done < <(wget -q -O- https://ethereum.github.io/solc-bin/bin/list.txt)
```

Comment:
Why don't you pipe this directly into grep?

Reply:
done